src/core_plugins/timelion/server/handlers/__tests__/parse_sheet.js

```diff
@@ -0,0 +1,23 @@
+const parseSheet = require('../lib/parse_sheet');
+
+const expect = require('chai').expect;
+
+describe('timelion parse_sheet function', function () {
```

Comment:
yes, test that \n and \r are now accepted.

but not sure if we already have tests in place. if not, this could be a separate effort.